source/install/self-managed-changelog.md

```diff
@@ -5,11 +5,120 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
 - [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
-- [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
+
+## Release v6.7 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-05-16**
+
+### Compatibility
+ - Updated Chrome recommended minimum version to v100+.
+
+### Important Upgrade Notes
+ - New schema changes were introduced in the form of a new index. The following notes the test results for the schema changes:
+    - MySQL 7M Posts - ~17s (Instance: db.r5.xlarge)
```

Comment:
To confirm, what are these times for? How long it takes to run the database queries?

Reply:
Pinging @agnivade to confirm

Reply:
Yes.